docs/cudf/source/_ext/PandasCompat.py

```diff
@@ -0,0 +1,151 @@
+# This file is adapted from official sphinx tutorial for `todo` extension:
+# https://www.sphinx-doc.org/en/master/development/tutorials/PandasCompat.html
+
+from docutils import nodes
+from docutils.parsers.rst import Directive
+from sphinx.locale import _
```

Comment:
Can we please alias this import as something sane... or just do what [sphinx.locale](https://www.sphinx-doc.org/en/master/_modules/sphinx/locale.html) actually does and construct this locally:
```suggestion
from sphinx.locale import get_translation
translator = get_translation('sphinx')  # Use this function instead.
```

Reply:
lol I was confused at `_` for a while too.